dspace/.htaccess

```diff
@@ -1,29 +1,39 @@
-RewriteEngine On
 
-# Rules controling links with the /vX.Y/ path segment
+AddType application/ld+json .jsonld
+
+/2024-1/RewriteEngine On
+
+# Rules controling links with the /vX.Y/ path segment pointing to IDSA
 ## Redirect main JSON-LD context
-RewriteRule ^(head|v[0-9][.][0-9])/context.json https://eclipse-dataspace-protocol-base.github.io/DataspaceProtocol/$1/common/schema/context.json [R=302,L,QSA]
+RewriteRule ^(head|v[0-9][.][0-9])/context.json https://international-data-spaces-association.github.io/ids-specification/$1/common/schema/context.json [R=302,L,QSA]
 
 ## Redirect JSON Schemas
-RewriteRule ^(head|v[0-9][.][0-9])/(catalog|negotiation|transfer)/([\w.-]+schema.json) https://eclipse-dataspace-protocol-base.github.io/DataspaceProtocol/$1/$2/message/schema/$3 [R=302,L,QSA]
-RewriteRule ^(head|v[0-9][.][0-9])/common/([\w.-]+schema.json) https://eclipse-dataspace-protocol-base.github.io/DataspaceProtocol/$1/common/schema/$2 [R=302,L,QSA]
+RewriteRule ^(head|v[0-9][.][0-9])/(catalog|negotiation|transfer)/([\w.-]+schema.json) https://international-data-spaces-association.github.io/ids-specification/$1/$2/message/schema/$3 [R=302,L,QSA]
+RewriteRule ^(head|v[0-9][.][0-9])/common/([\w.-]+schema.json) https://international-data-spaces-association.github.io/ids-specification/$1/common/schema/$2 [R=302,L,QSA]
 
 ## Redirect SHACL Shapes
-RewriteRule ^(head|v[0-9][.][0-9])/(catalog|negotiation|transfer)/([\w.-]+shapes?.ttl) https://eclipse-dataspace-protocol-base.github.io/DataspaceProtocol/$1/$2/message/shape/$3 [R=302,L,QSA]
-RewriteRule ^(head|v[0-9][.][0-9])/common/([\w.-]+shapes?.ttl) https://eclipse-dataspace-protocol-base.github.io/DataspaceProtocol/$1/common/shape/$2 [R=302,L,QSA]
+RewriteRule ^(head|v[0-9][.][0-9])/(catalog|negotiation|transfer)/([\w.-]+shapes?.ttl) https://international-data-spaces-association.github.io/ids-specification/$1/$2/message/shape/$3 [R=302,L,QSA]
+RewriteRule ^(head|v[0-9][.][0-9])/common/([\w.-]+shapes?.ttl) https://international-data-spaces-association.github.io/ids-specification/$1/common/shape/$2 [R=302,L,QSA]
 
-# Rules controling links with the /<year>/<number>/ path segment, e.g. /2024-1/
+# Rules controling links explicitly for release /2024-1/ pointing to IDSA
```

Comment:
The PR comment and this line has "2024-1/" but the rules are "2024/1/".  Same for "2025" below.

Reply:
This is intentional